openid-4-verifiable-presentations-1_0.md

```diff
@@ -460,6 +485,67 @@ To use `client_id_scheme` values `entity_id`, `did`, `verifier_attestation`, `x5
 
 Other specifications can define further values for the `client_id_scheme` parameter. It is RECOMMENDED to use collision-resistant names for such values.
 
+## Request URI Method POST {#request_uri_method_post}
+
+This request is handled by the Request URI endpoint of the Verifier.  
```

Comment:
this is a new endpoint, right? if so, I think that should be made clearer.

```suggestion
This request is handled by the Request URI endpoint of the Verifier.
```

Reply:
this endpoint was defined in RFC 9101. This spec just adds a new HTTP method to it.